azure/imageclient/client.go

```diff
@@ -0,0 +1,62 @@
+package imageclient
+
+import (
+	"encoding/xml"
+	"errors"
+	"fmt"
+
+	"github.com/MSOpenTech/azure-sdk-for-go/azure"
+)
+
+const (
+	azureImageListURL    = "services/images"
+	errInvalidImage      = "Can not find image %s in specified subscription, please specify another image name."
+	errParamNotSpecified = "Parameter %s is not specified."
+)
+
+//ImageClient is used to manage operations on Azure Locations
+type ImageClient struct {
+	client *azure.Client
+}
+
+//NewClient is used to instantiate a new ImageClient from an Azure client
+func NewClient(client *azure.Client) *ImageClient {
```

Comment:
What about moving all client sructs like ImageClient and HostedServiceClient to entities files? This way all classes will be in one place.